apps/v1beta1/groupversion_info.go

```diff
@@ -20,8 +20,8 @@ limitations under the License.
 package v1beta1
 
 import (
-	"github.com/openkruise/kruise-api/utils/scheme"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
```

Comment:
This file should not be modified

Reply:
the imported package is modified in the kruise repository